### DISCUSSION TITLE
CVEs - Golang specific

### DISCUSSION THREAD

**Discussion by williejay2017** [valkey-io/valkey#833] [Q&A]
I know you don't want security questions/discussions here but I have emailed and haven't received an acknowledgement 🫤. Can someone remedy the golang cve(s)?

**Comment by madolson** [comment]
@williejay2017 I just realized today that the email alias was misconfigured and everyone's emails are in a pending queue. I can't sort that out today, but you'll get a response shortly.

**Reply by madolson** [reply]
You should have a response now, apologies for the delay. New project, new woes.

**Reply by williejay2017** [reply]
Much appreciated, I totally understand.
